utils/test/testinput/gdas_smap2ioda.yaml

```diff
@@ -0,0 +1,7 @@
+provider: SMAP
+window begin: 2018-04-15T06:00:00Z
+window end: 2018-04-15T12:00:00Z
+output file: sss_smap.ioda.nc
+input files:
+- sss_smap_1.nc4
+#- sss_smap_2.nc4
```

Comment:
Un-comment L7, the concatenation should work.

Reply:
done